### ISSUE TITLE
UI-Design : Fonts & Sizes

### ISSUE THREAD

**Issue by dharinirajah** [Team-uMigrate/umigrate#174]
Put together a document highlighting multiple text style options w/ options for sizing and bolding. 

These fonts/sizes should be able to be visibly pleasing to the user on a mobile screen and should take into account accessibility for our users to be able to read if they struggle w/ reading.

List out what sizes you would you for various components of the mobile app & remockup the framer prototype w/ size options and font options (don't permanently adjust anything and work strictly on one page (that is not being updated by others). Please screenshot the options and put them in the document/method of presentation for the team to be able to discuss and highlight the pros and cons of each.
